nuget.config

```diff
@@ -10,4 +10,8 @@
 
     <add key="localpackages" value="localpackages" />
   </packageSources>
-</configuration>
\ No newline at end of file
+
+  <disabledPackageSources>
+    <clear />
```

Comment:
what does <clear /> do?  I was reading up on disabledPackgeSources, primary reference here:
https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#disabledpackagesources
and it refers to leaving the "list" empty.

Reply:
Also, I read some documentation that says this should go first and that is basically where it is in the WinUI3 nuget.config.

Reply:
Moved the disabledPackageSources block above the packageSources block as suggested offline